framework/tlvf/src/src/CmduMessageRx.cpp

```diff
@@ -19,18 +19,13 @@ CmduMessageRx::CmduMessageRx(uint8_t *buff, size_t buff_len) : CmduMessage(buff,
 
 CmduMessageRx::~CmduMessageRx() {}
 
-bool CmduMessageRx::parse(bool parse_tlvs)
+bool CmduMessageRx::parse()
 {
     msg.reset(true);
     auto cmduhdr = msg.addClass<cCmduHeader>();
     if (!cmduhdr)
         return false;
 
-    cmduhdr->class_swap(); //swap back cmduheader
```

Comment:
Was is swapped twice? How did it work before?

Reply:
Because `CmduMessage::getMessageType()` and `CmduMessage::getMessageId()` took care of swapping back. Now it is all aligned, swapping always happens and there is no special treatment to CmduHeader.